sycl/source/detail/pi.cpp

```diff
@@ -212,9 +212,11 @@ bool findPlugins(vector_class<std::pair<std::string, backend>> &PluginNames) {
   // search is done for libpi_opencl.so/pi_opencl.dll file in LD_LIBRARY_PATH
   // env only.
   //
-  PluginNames.push_back(std::make_pair<std::string, backend>(OPENCL_PLUGIN_NAME,
-                                                             backend::opencl));
-  PluginNames.push_back(
+  PluginNames.emplace_back(std::make_pair<std::string, backend>(
+      OPENCL_PLUGIN_NAME, backend::opencl));
+  PluginNames.emplace_back(std::make_pair<std::string, backend>(
+      LEVEL0_PLUGIN_NAME, backend::level0));
+  PluginNames.emplace_back(
```

Comment:
What is the problem with my previous proposal?
https://godbolt.org/z/VuYWZ8

Reply:
I made a mistake. Applied your proposal now.